Apple main site clone

jfariasf opened the pull request:
What was done:

Apple 2014 main site clone.

How was done:

- CSS and HTML

With the following features:
- CSS Gradients
- Clean use of Flex and Float
- Use of image-background

Description of the project:
https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-backgrounds-and-gradients

jfariasf committed:
Added navbar, body of the site and footer. Added CSS style and minor fixes

jfariasf committed:
Added relevant and descriptive content

Added the project information, features, technologies used, license and contact information.

jfariasf committed:
Changing wrong year of the original site

The original date of the site is 2014 not 1996

bolah2009 commented on index.html:
```diff
@@ -1,12 +1,232 @@
+                </form>
+                <!-- <div >
+
+                </div> -->
```
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

bolah2009 commented on index.html:
```diff
@@ -1,12 +1,232 @@
+                <li>iPhone</li>
+                <li>iPad</li>
+                <li>iTunes</li>
+                <li>Support</li>
```
Let's wrap all the list item with the `button` element or an anchor tag `<a>`.
Remember that to use `type="button"` if you are changing to `button` element to prevent the button from send data to an unexistent server.

bolah2009 commented:
# Awesome job @jfariasf :clap: :clap:

### [The clone](https://raw.githack.com/jfariasf/apple_gradients/abd90d455d29265fcc24a572e5bf02c2f6bc9113/index.html) looks good to me and uses CSS Backgrouds and Gradient effectively.

However, there are few changes before a :+1: 
- Nice job with a descriptive PR description and name. :1st_place_medal: 
  But this is not a `Milestone` project, how about changing it to the name of the project instead of a one-line description of what this PR will do to the master branch (Like a commit message). The description can also look like the readme: what was done, how it was done, a live link, the assignment link or/and even the source page link.
- Please, let's use `button` or anchor element for the navigation
- Please, use GitHub Flow in your next project.
- Add linter (`.stickler.yml`) config

#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

bolah2009 reviewed (changes requested):
# Awesome job @jfariasf :clap: :clap:

### [The clone](https://raw.githack.com/jfariasf/apple_gradients/abd90d455d29265fcc24a572e5bf02c2f6bc9113/index.html) looks good to me and uses CSS Backgrouds and Gradient effectively.

However, there are few changes before a :+1: 
- Nice job with a descriptive PR description and name. :1st_place_medal: 
  But this is not a `Milestone` project, how about changing it to the name of the project instead of a one-line description of what this PR will do to the master branch (Like a commit message). The description can also look like the readme: what was done, how it was done, a live link, the assignment link or/and even the source page link.
- Please, let's use `button` or anchor element for the navigation
- Please, use GitHub Flow in your next project.
- Add linter (`.stickler.yml`) config

#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

jfariasf committed:
Added a tag for the clickable menu. Removed a comment

jfariasf committed:
Merge branch 'dev' of https://github.com/jfariasf/apple_gradients into dev

jfariasf commented:
Pull update according to code review:

- Changed the Pull Request title and comment.
- Added the `a` tag of the clickable menu.
- Removed the comment.

The stickler files are in both the master and dev repos, I think you missed it:

- .stickler.yml 
- stylelint.config.js

By the way:

Please explain why Pull Request need to be similar to the README.md file, as in theory it should only describe the changes you are offering that comply with the main project. Otherwise I would feel like copy pasting the readme.

jfariasf committed:
Solving a small problem with text inside a tags

uche-inyama reviewed (changes requested):
## Hi jfariasf, good job :wave: 

Nice!, you have implemented the changes suggested by colleague, but there's one requirement left:

- [x] There is no linter checks on this project. [**Suggestion**] Findout if this project is turned on [stickler's site](https://stickler-ci.com/). 

 #### After implementing the change on stickler, with ```linter checks passed```  submit another form for code review.

##### ```Have a great day``` :slightly_smiling_face:

jfariasf committed:
Small change just to trigger stickler

bolah2009 commented:
#### I am leaving this message in case it could be of help.

### Setting up linter locally
An easy way to do this is to set up `[Stylelint](https://stylelint.io/)` locally:

   - run `npm install stylelint stylelint-config-recommended --save-dev`  (not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm))
   - copy [stylelint.config.js](./stylelint.config.js) to the root directory of your project
   - run `npx stylelint .` (there is a dot, also instead of the dot you can target the file you want to check e.g. `npx stylelint css/styles.css`
   - fix linter errors (Add the `--fix` flag to fix common errors like indentation, e.g `npx stylelint css/styles.css --fix` )

-----

### This is not a CRR

jfariasf committed:
Fixing linter suggestions

Converting tabs to spaces in the styles.css for stickler

Fixing more linter style problems

Fixing more linter css style issues

Fixing last style error linter

jfariasf commented:
Ok thanks to both of you. Indeed stickler wasn't enable on the repo.

Changes:

- Stickler activated and passing tests.

@bolah2009 That's tremendously helpful. Thanks!

Forison reviewed (commented):
# pull request under review by Addo

Forison reviewed (changes requested):
## Good Job @jfariasf 👋👋
#### You have successfully implemented the changes demanded by my colleagues.👋👋👋👋
your project looks good but lets us make it perfect by doing the following changes.🙂

- I have seen class and style attributes which is empty in your project those elements are highlighted by your linters take a look at them and remove them
> remember that inline styling is prohibited per the scope of the project. Just remove them since they are obsolete
- Another change required is that, there is a horizontal scroll on your page this could be due to poor width assignment
> try setting ```..page-width ``` to 100%; to have it respond as needed on different screen size
#### Kindly do those changes and submit for review, you are almost there.💪 
Good luck with your next submission🙂🙂🙂

jfariasf committed:
Removed empty attribute tags. Final commit

Linter fix

Linter fix

Linter fix
